index.html

```diff
@@ -29,8 +29,8 @@ <h4>Instant Save: ctrl + s </h4>
         window.addEventListener('DOMContentLoaded', (event) => {
             console.log('DOM fully loaded and parsed');
             fs.readFile('/note', 'utf8', function (err, data) {
-                if (err|| data.length === 0) {
-                    document.querySelector("#note").innerText = "Welcome to my note app!";
+                if (err || data.length === 0) {
+                    document.querySelector("#note").innerHTML = "Welcome to my note app!";
```

Comment:
Perhaps this should remain as innerText instead of innerHTML for proper encoding?

Reply:
Ye that might be a good idea